src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -94,13 +93,12 @@ void LoadARFF(const std::string& filename,
       else
       {
         throw std::runtime_error("unknown ARFF annotation '" + (*tok.begin()) +
-            "'");
+                                 "'");
```

Comment:
I have seen this a few times now... if the following line is double-indented, it's fine; it doesn't need to line up with the previous opening brace.